libs/components/src/lib/popup/popup.spec.ts

```diff
@@ -243,6 +243,18 @@ describe('vwc-popup', () => {
 			expect(element.open)
 				.toEqual(false);
 		});
+
+		it('should remove keydown listener after disconnection', async function() {
+			const spy = jest.fn();
+			element.addEventListener('keydown', spy);
+	
+			element.disconnectedCallback();
+			element.connectedCallback();
+	
+			element.dispatchEvent(new Event('keydown'));
+	
+			expect((spy as any).mock.calls.length).toEqual(1);
+		});
```

Comment:
I don't think this is a good test here.
What is your code supposed to do?
It `should remove keydown listener after disconnection`. But in this case, it does not remove the listener on the `element` itself - the listener is on the `anchor`.
So what you should test is that the listener is removed from the anchor and not from the element.

```suggestion
		it('should remove keydown listener after disconnection', async function() {
			const spy = jest.fn();
			anchor.addEventListener('keydown', spy);
	
			element.disconnectedCallback();
	
			anchor.dispatchEvent(new Event('keydown'));
	
			expect((spy as any).mock.calls.length).toEqual(0);
		});
```
In addition, you also need to test the case in which you change the `anchor`.